src/NodeApi/Runtime/NodejsPlatform.cs

```diff
@@ -25,25 +25,39 @@ public sealed class NodejsPlatform : IDisposable
     /// <summary>
     /// Initializes the Node.js platform.
     /// </summary>
-    /// <param name="libnodePath">Path to the `libnode` shared library, including extension.</param>
+    /// <param name="libnode">
+    /// Name of the `libnode` shared library.
+    /// Has to be a full file path when using .NET Framework.
+    /// </param>
     /// <param name="args">Optional application arguments.</param>
     /// <param name="execArgs">Optional platform options.</param>
     /// <exception cref="InvalidOperationException">A Node.js platform instance has already been
     /// loaded in the current process.</exception>
     public NodejsPlatform(
-        string libnodePath,
+        string libnode,
         string[]? args = null,
         string[]? execArgs = null)
     {
-        if (string.IsNullOrEmpty(libnodePath)) throw new ArgumentNullException(nameof(libnodePath));
-
         if (Current != null)
         {
             throw new InvalidOperationException(
                 "Only one Node.js platform instance per process is allowed.");
         }
 
-        nint libnodeHandle = NativeLibrary.Load(libnodePath);
+#if NET7_0_OR_GREATER
+        var entryAssembly = System.Reflection.Assembly.GetEntryAssembly();
+
+        nint libnodeHandle =
+            entryAssembly != null
+                ? NativeLibrary.Load(libnode, entryAssembly, null)
```

Comment:
It would involve figuring out the shared library extension for the platform (dylib/so/dll) and searching through LIB_PATH for any matching files, which might be a slightly intensive process, but I'd be happy to implement it.

I don't have a Windows machine to test it on though.

Reply:
I've implemented a partially-featured replica of the original method, but I can't test it because I don't have access to a Windows machine.